seed/java-sdk/seed.yml

```diff
@@ -41,27 +41,34 @@ fixtures:
     - customConfig: null
       outputFolder: no-custom-config
     - customConfig:
+        inline-path-parameters: true
         custom-dependencies:
           - "implementation jakarta.websocket:jakarta.websocket-api:2.1.0"
       outputFolder: custom-dependency
```

Comment:
This is fine for now since we're getting a ton of test coverage here, but we usually roll out a separate `customConfig` entry for individual test cases. So this was previously named `custom-dependency` since it only specifies `custom-dependencies`.

For example, we could have a separate `inline-path-parameters` that only sets `inline-path-parameters: true`.

Reply:
Gotcha, will do that next time!